Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -1939,29 +1941,15 @@ def prepare_DQM(self, sequence = 'DQMOffline'):
 		if 'HLT' in self.stepMap.keys() or self._options.hltProcess:
 			self.renameHLTprocessInSequence(sequence)
 
-		# if both HLT and DQM are run in the same process, schedule [HLT]DQM in an EndPath
-	        # not for fastsim
-		if 'HLT' in self.stepMap.keys() and not self._options.fast:
-			# need to put [HLT]DQM in an EndPath, to access the HLT trigger results
-			setattr(self.process,pathName, cms.EndPath( getattr(self.process, sequence ) ) )
-		else:
-			# schedule DQM as a standard Path
-			setattr(self.process,pathName, cms.Path( getattr(self.process, sequence) ) ) 
+                setattr(self.process,pathName, cms.EndPath( getattr(self.process, sequence ) ) )
 		self.schedule.append(getattr(self.process,pathName))
 
 	pathName='dqmofflineOnPAT_step'
 	for (i,sequence) in enumerate(postSequenceList):
                 if (i!=0):
                         pathName='dqmofflineOnPAT_%d_step'%(i)
 
-		# if both MINIAOD and DQM are run in the same process, schedule DQM in an EndPath
-		if 'PAT' in self.stepMap.keys():
-			# need to put DQM in an EndPath, to access the miniAOD filter results
-			setattr(self.process,pathName, cms.EndPath( getattr(self.process, sequence ) ) )
-		else:
-			# schedule DQM as a standard Path
-			setattr(self.process,pathName, cms.Path( getattr(self.process, sequence) ) )
-		self.schedule.append(getattr(self.process,pathName))
```

Comment:
I suspect this removal caused loss of MiniAOD validation histograms in 810pre12, see
https://hypernews.cern.ch/HyperNews/CMS/get/relval/6213/46/2.html

I have a proposal for a fix in #16265.